python/setup.py

```diff
@@ -99,6 +99,18 @@ def config_cython():
         return []
 
 
+def compile_summary_protobuf():
+    proto_path = '../python/mxnet/contrib/summary/proto'
+    proto_files = os.path.join(proto_path, '*.proto')
+    cmd = 'protoc ' + proto_files + ' --python_out=' + proto_path + ' --proto_path=' + proto_path
+    return os.system(cmd)
+
+
+if compile_summary_protobuf() != 0:
```

Comment:
@marcoabreu Does CI run this file for installing the MXNet python package? The current unit tests are failing because the protobuf files are not compiled into `.py` files as indicated in this file before installation. How could I configure the CI to compile the protobuf files first?

Reply:
No it doesn't.

Reply:
As explained offline, we're not running setup.py as part of our CI pipeline - instead, we're defining the path to the mxnet lib as an environment variable. I'd appreciate it you could integrate setup.py and remove that env var.